edx_exams/apps/lti/api.py

```diff
@@ -1,59 +1,3 @@
 """
```

Comment:
do we need this file at all anymore?

Reply:
We don't. I believe `api.py` and `models.py` are autogenerated by the cookie cutter. I can delete them.